src/components/main/person-request/PersonRequestList.vue

```diff
@@ -40,7 +40,11 @@
           cancelButtonText : 'Cancel',
           type             : 'warning'
         })
-        .then(() => personRequestService.delete(personRequest._id))
+        .then(() => {
+          personRequestService.delete(personRequest._id).then(() => {
```

Comment:
why going to `person-request-list` we are already in this route, if you want the person request to be removed from the table just splice the the array, like we are doing in person list.